Agents/MackayCalculatorAgent/readme.md

```diff
@@ -0,0 +1,61 @@
+##MacKay Calculator Agent
+This agent is designed to provide access to the controls and outputs (I/O) of the Singapore MacKay Carbon Calculator model which is originally in Excel format. The agent interacts with the worldavatar knowledge graph with APIs to automatically update the calculator with values from the KG in real time.
+
+A React web interface is additionally provided to interact with the Singapore MacKay Calculator.
+
+
+
+##API
+
+###Endpoints
+####Set Calculator levers and retrieve data
+URL:  /data 
+
+Method: POST
+
+Data constraints: Provide lever values:
+
+`````{"levers":[ an array of 45 intergers in range 1-4]}`````
+
+
+
+Success Responses: Content: 
+
+```{"status": "Success", "values": [...]}```
+
+
+#### Update the Calculator with World Avatar KG
+URL: /update
+
+Method: Get
+
+Success Responses: Code: 200 OK
+
+
+##  Usage
+###Overview
+Currently the agent relies on an Excel model and therefore requires a Windows OS and a copy of the Excel software to run. Until we finish converting the model into another form of code in the future this agent will not have a dockerized version.
+
+###Requirements
+python >=3.10, java jdk>=11, nodeJS>=18
+
+###Setup
+
+
+1. Install required python packages.
+```shell
+pip install -r requirements.txt
+```
+2. Download Mackay Calculator Excel model and mapping csvs (```controls.csv```, ```output.csv```,```single_values.csv```) into ```model/```.
```

Comment:
Please add a hint where to get these from. A good location could be `dropbox/IRP3 CAPRICORN shared folder/_TWA_Shared_Data/`.

Reply:
I have put the files in Dropbox and add a link to README.